doc/DataSlice.md

```diff
@@ -17,11 +17,14 @@ Below is the metadata list and a note explaining how they are calculated.
 
     * `total_rows` (`int`): total rows returned by a user query.
     * `total_returned_rows` (`int`): total of rows returned in a slice of results.
-    * `execution_time` (`float`): in milliseconds, query execution time.
-    * `conversion_time` (`float`): in milliseconds, time to convert results in a `pandas.DataFrame`.
+    * `query_generation_time` (`float`): in seconds, the time to modify/create a query to get just the slice and start the process to request external data.
+    * `data_extraction_time` (`float`): in seconds, the time to request the result of the modified query from external service. 
+    * `data_conversion_time` (`float`): in seconds, the time to convert results in a `pandas.DataFrame`.
+    * `permission_application_time` (`float`): in seconds, the time to filter datas with permissions.
```

Comment:
In the http data service, it's called `data_filtered_from_permission_time`. Which one do you want to keep? (the important thing is that it's the same everywhere).

Reply:
@Sanix-Darker If you prefer, you can revert my proposal.

Reply:
👍🏾